data_scientist_skills/data.py

```diff
@@ -3,14 +3,18 @@
 from nltk.corpus import stopwords
 from nltk.stem import WordNetLemmatizer
 import string
-import unidecode
+import unidecode  #This import us unused do we need to have this? -JP
```

Comment:
Used for removing accents in data.clean(). I've had a lot of trouble with this package, somedays it works, somedays it doesn't